tfgrid3/umbrel/dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    UMBREL_VERSION="release" \
+    UMBREL_REPO="getumbrel/umbrel" \
+    UMBREL_INSTALL_PATH="/umbrel" 
+
+RUN apt update && \
+    apt -y install wget curl vim net-tools iputils-ping openssh-server docker.io \
+    fswatch jq rsync sudo iproute2 git gettext-base python3 gnupg avahi-daemon avahi-discover libnss-mdns nginx ufw
+
+RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
+    chmod +x /usr/local/bin/docker-compose
+
+RUN mkdir -p ${UMBREL_INSTALL_PATH}; \
+    mkdir -p /etc/zinit;
+COPY ./scripts /scripts
+RUN chmod -R +x /scripts; 
+RUN /bin/bash -c  "/scripts/yq.sh;"
+
+
+RUN version=$(get_umbrel_version); \
+    curl --location "https://api.github.com/repos/${UMBREL_REPO}/tarball/${version}" | \
+    tar --extract --gzip --strip-components=1 --directory="${UMBREL_INSTALL_PATH}"; \
+    sed --i '/--detach/,${s// /;b};$q1' ${UMBREL_INSTALL_PATH}/scripts/start || echo faild;
```

Comment:
Expressions don't expand in single quotes, did you mean to use double quotes for sed statement?

Reply:
It works fine, as I need to return error if the `--detach` is not exist, check this [linl](https://stackoverflow.com/questions/15965073/return-value-of-sed-for-no-match)
any way i will remove it as there are many changes in the image